packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -292,6 +292,7 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
 			socketReference,
 			telemetryLogger,
 			enableMultiplexing,
+			uuid(),
```

Comment:
Given it's generated on the fly, I'd rather do it inside of the object (similar how we do so for Container).